rfc8446.xml

```diff
@@ -2841,12 +3178,11 @@ attackers could obtain a signature of a message with a chosen 32-byte
 prefix (ClientHello.random). The initial 64-byte pad clears that prefix
 along with the server-controlled ServerHello.random.</t>
 
-<t>The context string for a server signature is:
-“TLS 1.3, server CertificateVerify”
-The context string for a client signature is:
-“TLS 1.3, client CertificateVerify”
-It is used to provide separation between signatures made in different
-contexts, helping against potential cross-protocol attacks.</t>
+<t>The context string for a server signature is "TLS 1.3,
+server CertificateVerify".  The context string for a client signature is
+"TLS&nbsp;1.3, client CertificateVerify".  It is used to provide
```

Comment:
We weren't sure what we could do about this, because the original text was on multiple lines.  Dragged and dropped a snapshot file here; if it doesn't display properly, please see <https://www.rfc-editor.org/authors/rfc8446-regarding-de-break-request.png>.

The cyan window shows the excerpt from the original approved XML copy (i.e., the file that was provided to us when we added this document to our queue in late March).  Please note that the original XML file arrived with many line breaks; were the line breaks not there on your end and somehow introduced during our normal upload process?

The dark blue window shows the first iteration (which prompted your request to de-break the lines).

The lavender window shows the latest iteration of the XML file with line breaks per the original.